lambdas/src/SmartContentServerFetcher.ts

```diff
@@ -1,28 +1,28 @@
 import log4js from "log4js"
-import fetch from "node-fetch"
```

Comment:
Can we remove the dependency to `node-fetch` from the project now? I mean the bazel file, not the package json

Reply:
Not yet, I preferred to not change the content-v2 translator (that uses node-fetch).